### ISSUE TITLE
IOError during pip install of zhmcclient-0.7.0.tar.gz on CentOS 7.3

### ISSUE THREAD

**Issue by leopoldjuergen** [zhmcclient/python-zhmcclient#127]
### Actual behavior
Get 'IOError: [Errno 2] No such file or directory: 'README.md'' when trying to do a pip install.

### Expected behavior
No error.

### Execution environment
Used centos/7 image based on CentOS Linux 7.3.1611 in a VirtualBox/Vagrant environment.
Steps to reproduce issue:
```
$ vagrant box add centos/7
```
Used Vagrantfile:
```
# -*- mode: ruby -*-
# vi: set ft=ruby :

# Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
VAGRANTFILE_API_VERSION = "2"

Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
  config.vm.box = "centos/7"
```
```
$ vagrant up
$ vagrant ssh

$ sudo yum install -y git
$ sudo yum install -y python-virtualenv

$ virtualenv venv
$ source venv/bin/activate

(venv)[vagrant@localhost ~]$ pip install zhmcclient
Downloading/unpacking zhmcclient
  Downloading zhmcclient-0.7.0.tar.gz (110kB): 110kB downloaded
  Running setup.py egg_info for package zhmcclient
    
    Installed /home/vagrant/venv/build/zhmcclient/pbr-1.10.0-py2.7.egg
    [pbr] Processing SOURCES.txt
    warning: LocalManifestMaker: standard file '-c' not found
    
    warning: no previously-included files found matching '.gitignore'
    warning: no previously-included files found matching '.gitreview'
    warning: no previously-included files matching '*.pyc' found anywhere in distribution
Downloading/unpacking six (from zhmcclient)
  Downloading six-1.10.0.tar.gz
  Running setup.py egg_info for package six
    
    no previously-included directories found matching 'documentation/_build'
Downloading/unpacking pbr>=1.8 (from zhmcclient)
  Downloading pbr-1.10.0.tar.gz (111kB): 111kB downloaded
  Running setup.py egg_info for package pbr
    [pbr] Processing SOURCES.txt
    warning: LocalManifestMaker: standard file '-c' not found
    
    warning: no previously-included files found matching '.gitignore'
    warning: no previously-included files found matching '.gitreview'
    warning: no previously-included files matching '*.pyc' found anywhere in distribution
Downloading/unpacking requests>=2.10.0 (from zhmcclient)
  Downloading requests-2.12.4.tar.gz (547kB): 547kB downloaded
  Running setup.py egg_info for package requests
    
    warning: no files found matching 'test_requests.py'
Downloading/unpacking decorator (from zhmcclient)
  Downloading decorator-4.0.10.tar.gz (68kB): 68kB downloaded
  Running setup.py egg_info for package decorator
    
Downloading/unpacking tabulate (from zhmcclient)
  Downloading tabulate-0.7.7.tar.gz
  Running setup.py egg_info for package tabulate
    
Downloading/unpacking progressbar2 (from zhmcclient)
  Downloading progressbar2-3.12.0.tar.gz
  Running setup.py egg_info for package progressbar2
    your setuptools is too old (<12)
    setuptools_scm functionality is degraded
    zip_safe flag not set; analyzing archive contents...
    
    Installed /home/vagrant/venv/build/progressbar2/pytest_runner-2.9-py2.7.egg
    
    warning: no files found matching 'AUTHORS.rst'
    warning: no files found matching 'README.txt'
Downloading/unpacking click (from zhmcclient)
  Downloading click-6.6.tar.gz (283kB): 283kB downloaded
  Running setup.py egg_info for package click
    
    warning: no previously-included files matching '*.pyc' found under directory 'docs'
    warning: no previously-included files matching '*.pyo' found under directory 'docs'
    warning: no previously-included files matching '*.pyc' found under directory 'tests'
    warning: no previously-included files matching '*.pyo' found under directory 'tests'
    warning: no previously-included files matching '*.pyc' found under directory 'examples'
    warning: no previously-included files matching '*.pyo' found under directory 'examples'
    no previously-included directories found matching 'docs/_build'
Downloading/unpacking click-spinner==0.1.3 (from zhmcclient)
  Downloading click_spinner-0.1.3.tar.gz
  Running setup.py egg_info for package click-spinner
    Traceback (most recent call last):
      File "<string>", line 16, in <module>
      File "/home/vagrant/venv/build/click-spinner/setup.py", line 5, in <module>
        with open('README.md') as f:
    IOError: [Errno 2] No such file or directory: 'README.md'
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):

  File "<string>", line 16, in <module>

  File "/home/vagrant/venv/build/click-spinner/setup.py", line 5, in <module>

    with open('README.md') as f:

IOError: [Errno 2] No such file or directory: 'README.md'
```

* zhmcclient version: zhmcclient-0.7.0.tar.gz
* Operating system (type+version): CentOS Linux 7.3.1611

**Comment by leopoldjuergen** [comment]
Not seen any more with version 0.8.0.

**Comment by andy-maier** [comment]
This is an error in click-spinner that was fixed in click-spinner v0.1.5. Zhmcclient removed the pinning of the click-spinner version in a commit that was merged into zhmcclient v0.8.0.
